### PR TITLE
add location to user's profile

### PR DESCRIPTION
**Pull request by ngtrduc**

### CODE REVIEW

File: db/migrate/20160225015453_create_profiles.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ def change
       t.integer :programming_language_id
       t.integer :user_progress_id
       t.integer :status_id
+      t.integer :location_id, index: true, foreign_key: true
```
Comment (NgengChhengkim) [inline db/migrate/20160225015453_create_profiles.rb]: dung reference

File: db/migrate/20160524024845_create_locations.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@ class CreateLocations < ActiveRecord::Migration
   def change
     create_table :locations do |t|
       t.string :name
+      t.integer :user_id
```
Comment (NgengChhengkim) [inline db/migrate/20160524024845_create_locations.rb]: dung reference from User

File: app/models/user.rb [hunk 3]
```diff
@@ -35,11 +35,11 @@ class User < ApplicationRecord
     :id, :start_training_date, :leave_date, :finish_training_date,
     :ready_for_project, :contract_date, :naitei_company, :trainer_id,
     :user_type_id, :university_id, :programming_language_id, :user_progress_id,
-    :status_id
+    :status_id, :location_id
   ]
```
Comment (NgengChhengkim) [inline app/models/user.rb]: user has on location

**Comment by qtsoul95** [comment]
ok

**Comment by NgengChhengkim** [comment]
OK

**Pull request merged**